### ISSUE TITLE
Launch a trademark working group

### ISSUE THREAD

**Issue by ehuss** [rust-lang/leadership-council#30]
We should formally create a group to resolve the trademark policy.

**Comment by jonathanpallant** [comment]
I think we did this? It's currently me and @m-ou-se.

**Comment by ehuss** [comment]
I don't think it was formalized in a sense that a team was created in the team database, or minimally, have a committee recorded in https://github.com/rust-lang/leadership-council/tree/main/committees.

**Comment by ehuss** [comment]
I'm going to close as I think the trademark issues have largely been resolved. With the final reworked policy at https://rustfoundation.org/policy/rust-trademark-policy/.

There were previous updates in:
https://blog.rust-lang.org/2024/11/06/trademark-update/
https://rustfoundation.org/media/rust-trademark-policy-updates/

And the final announcement was https://rustfoundation.org/media/rust-language-trademark-policy-updates-explained/
